py/src/call.py

```diff
@@ -127,9 +131,16 @@ def loop_inner(connection, command):
     if not check_schema(connection):
         raise UnexpectedSchemaVersion
 
-    (block_info, global_root) = resolve_block(connection, command["at_block"])
+    # default to "call" in case I forget to fix up old tests and make this
+    # required
+    verb = command["command"] if "command" in command else "call"
```

Comment:
Do we still need this default? It'd be great to make command required.

Reply:
Done in b3e9f232eb34d4a051a802355620b66b46e283a9